pkg/component/logging/fluentoperator/customresources/cluster_outputs.go

```diff
@@ -94,7 +94,6 @@ SendDeletedClustersLogsToDefaultClient true
 CleanExpiredClientsPeriod 1h
 ControllerSyncTimeout 120s
 PreservedLabels origin,namespace_name,pod_name
-TenantID operator
```

Comment:
As we talked offline, this change prevents users/operators to fetch old logs from plutono. This change deserves a release note of type `breaking user`.